MicrosoftTeams.schema.json

```diff
@@ -10,23 +10,58 @@
         "manifestVersion": {
             "type": "string",
             "description": "The version of the schema this manifest is using.",
-            "const": "1.4"
+            "const": "1.5"
         },
         "version": {
             "type": "string",
             "description": "The version of the app. Changes to your manifest should cause a version change. This version string must follow the semver standard (http://semver.org).",
             "maxLength": 256
         },
         "id": {
-            "type": "string",
-            "description": "A unique identifier for this app. This id must be a GUID.",
-            "pattern": "^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$"
+            "$ref": "#/definitions/guid",
+            "description": "A unique identifier for this app. This id must be a GUID."
         },
         "packageName": {
             "type": "string",
             "description": "A unique identifier for this app in reverse domain notation. E.g: com.example.myapp",
             "maxLength": 64
         },
+        "localizationInfo": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "defaultLanguageTag": {
+                    "$ref": "#/definitions/languageTag",
+                    "description": "The language tag of the strings in this top level manifest file.",
+                    "default": "en-us"
+                },
+                "additionalLanguages": {
+                    "type": "array",
+                    "uniqueItems": true,
+                    "items": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "properties": {
+                            "languageTag": {
+                                "$ref": "#/definitions/languageTag",
+                                "description": "The language tag of the strings in the provided file."
```

Comment:
we should either here or via documentation add a link that describes the language codes we support.